src/seed/unseed-db.js

```diff
@@ -0,0 +1,27 @@
+import { query } from '../graphql-schema'
+import { v1 as neo4j } from 'neo4j-driver'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const driver = neo4j.driver(
+    process.env.NEO4J_URI || 'bolt://localhost:7687',
+    neo4j.auth.basic(
+        process.env.NEO4J_USER || 'neo4j',
+        process.env.NEO4J_PASSWORD || 'neo4j'
+    )
+)
+let session = null
+session = driver.session()
+query('MATCH (n) OPTIONAL MATCH (n)-[r]-() DELETE n,r', session).then(() => {
```

Comment:
you could just use `MATCH (n) DETACH DELETE n` which is more readable.
```suggestion
query('MATCH (n) DETACH DELETE n', session).then(() => {
```

Reply:
Also we should check if the app is NOT running in production mode!

Reply:
will be changed